.github/workflows/wheel_linux_x86_64.yml

```diff
@@ -216,5 +216,5 @@ jobs:
         uses: pypa/gh-action-pypi-publish@release/v1
         with:
           user: __token__
-          password: ${{ matrix.pl_backend == "lightning_qubit" && secrets.TEST_PYPI_API_TOKEN || secrets.TEST_PYPI_LKOKKOS_TOKEN }}
+          password: ${{ (matrix.pl_backend == 'lightning_qubit') && secrets.TEST_PYPI_API_TOKEN || secrets.TEST_PYPI_LKOKKOS_TOKEN }}
```

Comment:
Wow didn't know this worked in github actions, it's similar to the Python behaviour I suppose where it produces one of the arguments rather than a Boolean?

Reply:
Yep, exactly that: https://docs.github.com/en/actions/learn-github-actions/expressions#example